tasks/deployment/grants-program-proposal.ts

```diff
@@ -0,0 +1,15 @@
+import { types } from 'hardhat/config';
+
+import mainnetAddresses from '../../src/deployed-addresses/mainnet.json';
+import { hardhatTask } from '../../src/hre';
+import { createGrantsProgramProposal } from '../../src/migrations/grants-program-proposal';
+
+hardhatTask('deploy:grants-program-proposal', 'Create proposal to Launch DGP with multisig funding.')
+    .addParam('proposalIpfsHashHex', 'IPFS hash for the uploaded DIP describing the proposal', undefined, types.string)
```

Comment:
Do you want to add the encoded IPFS hash as a default parameter? https://github.com/dydxfoundation/dip/blob/master/content/ipfs-dips/DIP-6-Ipfs-hashes.json#L4